tutorials/observing_examples.ipynb

```diff
@@ -0,0 +1,372 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from synphot import Observation\n",
+    "import numpy as np\n",
+    "\n",
+    "obs = Observation(photlam_hatp11, agile)\n",
```

Comment:
Here, assuming that the throughput file is binned correctly already, you can pass in `binset=agile.waveset`. Alternately, you may also inquire with APO for the values, and then use `numpy` to construct them as array before passing it into `binset`.

For example, here is a `binset` file for HST/COS using one of its grism filter: [cos_fuv_g130m_1291.txt](https://github.com/spacetelescope/stsynphot_refactor/blob/master/stsynphot/data/wavecats/cos_fuv_g130m_1291.txt)